sdk/search/azure-search-documents/azure/search/documents/indexes/models/_index.py

```diff
@@ -681,6 +683,8 @@ def pack_search_field(search_field: SearchField) -> _SearchField:
         synonym_map_names = search_field.get("synonym_map_names")
         fields = search_field.get("fields")
         fields = [pack_search_field(x) for x in fields] if fields else None
+        vector_search_dimensions = search_field.get("vector_search_dimensions")
+        vector_search_profile = search_field.get("vector_search_profile")
```

Comment:
will this change end up being a breaking change or just internal rename ?

Reply:
Yes. You find it. :)

This is a breaking change service introduced. The good news is that this is a  beta feature hence it is not breaking against last GA.

I would like to break it soon than later.

Reply:
lol :) awesome, then other than a nit mention in the beta changelog, it looks fine